css-brochure-form/styles.css

```diff
@@ -0,0 +1,67 @@
+h2 {
+  text-transform: uppercase;
+  text-align: center;
+  font-family: 'Brandon Grotesque', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  src: url('BrandonGrotesque-Medium.woff2') format('woff2');
+  color: rgb(223, 67, 5);
+  font-size: 16px;
+}
+
+.headings {
+  text-transform: uppercase;
+  font-family: 'Brandon Grotesque', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  src: url('BrandonGrotesque-Medium.woff2') format('woff2');
+  font-size: 12px;
+  padding-left: 3%;
+}
+
+form {
+  justify-content: center;
+  margin: auto;
+  width: auto;
+  padding-left: 30%;
+  padding-right: 30%;
```

Comment:
These can be removed.

Reply:
If you were using this to center the form, you can simply apply a width to the form, and the `margin: auto` will use the extra space to center the element 👍